dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp

```diff
@@ -1,74 +1,160 @@
+                    <button
+                        dojoType="dijit.form.Button"
+                        dojoAttachEvent="onClick:_clearSearch"
+                        iconClass="cancelIcon"
+                        class="dijitButtonFlat"
+                        style="margin-top: 16px"
+                    >
+                        <%= LanguageUtil.get(pageContext, "Clear") %>
+                    </button>
                 </div>
             </div>
         </div>
-
-	   <!-- START Right Column -->
-        <div dojoType="dijit.layout.ContentPane" splitter="true" region="center" class="portlet-main-wrapper">
-		   <div dojoAttachPoint="contentWrapper">
-		       <div class="portlet-toolbar">
-		           <div dojoAttachPoint="matchingResultsDiv" class="portlet-toolbar__matching-results" style="visibility: hidden"><%= LanguageUtil.get(pageContext, "Results") %></div>
-		           <div id='addContentTypeDropdown'></div>
-		           <div dojoAttachPoint="addContentletButton" class="portlet-toolbar__add-contentlet" style="display: none">
-		               <button dojoType="dijit.form.Button" onClick="addNewContentlet()"><%= LanguageUtil.get(pageContext, "Add-New-Content") %></button>
-		           </div>
-		           <div dojoAttachPoint="relateDiv" id="doRelateContainer">
-		               <button dojoType="dijit.form.Button" dojoAttachEvent='onClick:_doRelateContent' iconClass="searchIcon"><%= LanguageUtil.get(pageContext, "Relate") %></button>
-		           </div>
-		       </div>
-		       <table dojoAttachPoint="results_table" class="listingTable relateContent"></table>
-		   </div>
-		   <div class="portlet-pagination">
-		       <div dojoAttachPoint="previousDiv" style="display: none;">
-		           <button dojoType="dijit.form.Button" class="bg" dojoAttachEvent='onClick:_previousPage' iconClass="previousIcon"><%= LanguageUtil.get(pageContext, "Previous") %></button>
-		       </div>
-		       <div></div>
-		       <div dojoAttachPoint="nextDiv" style="display: none;">
-		           <button dojoType="dijit.form.Button" class="bg" dojoAttachEvent='onClick:_nextPage' iconClass="nextIcon"><%= LanguageUtil.get(pageContext, "Next") %></button>
-		       </div>
-		   </div>
-	
-	    </div>
-
-</div>
-    </form>
-
+    </div>
+    <!-- START Right Column -->
+    <div dojoType="dijit.layout.ContentPane" class="portlet-main-wrapper">
+        <div dojoAttachPoint="contentWrapper">
+            <div class="portlet-toolbar">
+                <div
+                    dojoAttachPoint="matchingResultsDiv"
+                    class="portlet-toolbar__matching-results"
+                    style="visibility: hidden"
+                >
+                    <%= LanguageUtil.get(pageContext, "Results") %>
+                </div>
+                <div id="addContentTypeDropdown"></div>
+                <div
+                    dojoAttachPoint="addContentletButton"
+                    class="portlet-toolbar__add-contentlet"

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5084653)